Library/Formula/ola.rb

```diff
@@ -3,6 +3,7 @@ class Ola < Formula
   homepage "https://www.openlighting.org/ola/"
   url "https://github.com/OpenLightingProject/ola/releases/download/0.10.0/ola-0.10.0.tar.gz"
   sha256 "cae8131a62f0ff78d399c42e64a51b610d7cf090b606704081ec9dd2cf979883"
+  revision 2
```

Comment:
Don't need the revision unless we're making a breaking change here. Also would be `revision 1`; default is `0` and unwritten.